src/MOI/MOI_wrapper.jl

```diff
@@ -1672,6 +1679,20 @@ function MOI.get(
     return MOI.Interval(lower, upper)
 end
 
+function MOI.set(
+    model::Optimizer,
+    ::MOI.ConstraintSet,
+    c::MOI.ConstraintIndex{MOI.VariableIndex, MOI.EqualTo{Float64}},
+    s::MOI.EqualTo{Float64}
+)
+    MOI.throw_if_not_valid(model, c)
+    lower, upper = _bounds(s)
+    @assert lower == upper
+    info = _info(model, c)
+    _fix_variable_value(model, info, lower)
```

Comment:
Instead of introducing this new function, just put the `XPRSchgbounds` call here directly.

Also work a comment that `XPRSchgbounds` expects vectors, so we pass singular `Ref`s, and that `B` stands for "both" bounds.

Reply:
Fixed via [24c98d4](https://github.com/jump-dev/Xpress.jl/pull/185/commits/24c98d4f7cd0b4af16917273092d7896060e7379)